docs/learn/universal-profile/metadata/edit-profile.md

```diff
@@ -129,98 +78,68 @@ Use the JSON template file below, and follow **steps 1.1 and 1.2** to fill the b
-
-:::
-
-![ipfs LUKSO settings](../img/ipfs-lukso-settings.jpg)
-
-Drag & Drop your images (you can upload multiple images at once) and _upload_ them. Once the process is completed:
-
-1. Copy the IPFS file identifier (`CID`) shown in the `hash` field marked with green below.
-2. Paste the `CID` into the `url` field in our JSON file, beginning with `ipfs://` at the start.
-
-![ipfs file upload](../img/ipfs-file-upload.jpg)
-
-Make sure to save your JSON file after you have added all your details and images.
-
-:::note
-Don't forget to delete the comments in the JSON file!
-:::
-
-We are now ready to apply these changes to our Universal Profile. We will see how in the next section :arrow_down:
-
-## Step 2 - Upload the JSON file to IPFS
+:::caution
+Image sizes should be written as numbers, not as strings.
 
-:::note Notice
-You should do the rest of this tutorial should be done in a **new file** (`main.js`).
+The **max image width** supported by [universalprofile.cloud](https://universalprofile.cloud) is: `profileImage <= 800px`, `backgroundImage <= 1800px`
 :::
 
-We will now start writing the main code of the tutorial.
-Create a new file, `main.js`.
+We are now ready to apply these changes to our Universal Profile. We will see how in the next section. :arrow_down:
 
-<!-- TODO: don't use lsp-factory anymore-->
+## Upload the JSON file to IPFS
 
 ```javascript title="main.js"
-import { LSPFactory } from '@lukso/lsp-factory.js';
+import { IPFSHttpClientUploader } from '@lukso/data-provider-ipfs-http-client';
 // reference to the previously created JSON file (LSP3Profile metadata)
-import jsonFile from './UniversalProfileMetadata.json';
+import jsonFile from './LSP3Metadata.json';
 
-const provider = 'https://rpc.testnet.lukso.network'; // RPC provider url
+const ipfsProvider = new IPFSHttpClientUploader(
+  'http://127.0.0.1:5001/api/v0/add',
+);
 
-const lspFactory = new LSPFactory(provider, {
-  deployKey: PRIVATE_KEY,
-  chainId: 2828, // Chain Id of the network you want to deploy to
-});
+const provider = 'https://4201.rpc.thirdweb.com'; // RPC provider url
 
 async function editProfileInfo() {
   // Step 2 - Upload our JSON file to IPFS
-  const uploadResult = await lspFactory.UniversalProfile.uploadProfileData(
+  const { lsp3ProfileIPFSUrl, lsp3ProfileIPFSHash } = await ipfsProvider.upload(
     jsonFile.LSP3Profile,
   );
-  const lsp3ProfileIPFSUrl = uploadResult.url;
-  // ipfs://Qm...
 }
 ```
 
-## Step 3 - Setup erc725.js and encode the LSP3Profile data
+## Setup erc725.js and encode the LSP3Profile data
 
 The next step is to **prepare the data** used to edit our Universal Profile. _Preparing the data_ means **encoding it** to write in on our Universal Profile ERC725Y smart contract.
 
```

Comment:
For the content below, suggest make it a one liner:

From this:

<img width="1334" alt="image" src="https://github.com/user-attachments/assets/1cba173f-4667-492b-9f9a-28352883d5f6">

To this:

To do so, we will use our [erc725.js] library, which helps us encode the data easily. We will need the following: